src/components/Themes/Themes.tsx

```diff
@@ -0,0 +1,84 @@
+import {ArrowUpFromSquare} from '@gravity-ui/icons';
+import {Col, Grid, Row} from '@gravity-ui/page-constructor';
+import {Button, Flex, Icon, Text, ThemeProvider} from '@gravity-ui/uikit';
+import {useTranslation} from 'next-i18next';
+import React, {useCallback, useMemo, useState} from 'react';
+
+import {DEFAULT_THEME} from '../../constants';
+import {block} from '../../utils';
+import {TagItem, Tags} from '../Tags/Tags';
+
+import {ThemeSection} from './ThemeSection/ThemeSection';
+import './Themes.scss';
+import {ThemeSectionType} from './types';
+
+const b = block('themes');
+
+export const Themes = () => {
+    const {t} = useTranslation('themes');
+
+    const tags: TagItem[] = useMemo(
+        () => [
+            {
+                value: ThemeSectionType.Colors,
+                title: t('tags_colors'),
+            },
+            {
+                value: ThemeSectionType.Typography,
+                title: t('tags_typography'),
+            },
+            {
+                value: ThemeSectionType.BorderRadius,
+                title: t('tags_borderRadius'),
+            },
+            {
+                value: ThemeSectionType.Preview,
+                title: t('tags_preview'),
+            },
+        ],
+        [t],
+    );
+
+    const [activeTag, setActiveTag] = useState<string>(tags[0].value);
+
+    const onExportButtonClick = useCallback(() => {
+        //TODO add logic here
+    }, []);
+
+    return (
+        <Grid className={b()}>
+            <Row className={b('title')}>
+                <Col sizes={12}>
+                    <Text className={b('title__text')}>{t('title')}</Text>
+                </Col>
+            </Row>
+            <Flex className={b('header-actions')}>
+                <Tags
+                    className={b('tabs')}
+                    items={tags}
+                    value={activeTag}
+                    onChange={setActiveTag}
+                />
+
+                <Button
+                    className={b('export-theme-btn')}
```

Comment:
In the design, the export button tag has a different border radius.

<img width="447" alt="image" src="https://github.com/gravity-ui/landing/assets/13693437/a4902269-8df1-4e7f-90a6-b430f6f59d7a">

Reply:
all XL btn in landing has bigger border radius, i will go to Yuri and ask him about it